{
  final ListIterator iterator=range.listIterator(0);
  assertFalse("iterator has next value",iterator.hasNext());
  assertFalse("iterator has previous value",iterator.hasPrevious());
  try {
    range.listIterator(1);
    fail("got list iterator at index 1");
  }
 catch (  IndexOutOfBoundsException e) {
    assertTrue("expected exception thrown",true);
  }
}
